src/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit.csproj

```diff
@@ -99,4 +140,63 @@
   
   <Import Project="$(MSBuildSDKExtrasTargets)" Condition="Exists('$(MSBuildSDKExtrasTargets)')" />
   
+
+  <UsingTask TaskName="MakeAndroidResourcesInternalTask" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v4.0.dll" Condition="'$(TargetFramework)' == '$(AndroidTargetFramework)'">
```

Comment:
This makes the auto-generated resources internal

Reply:
Move it all to a `.targets` file in the Android folder and include it from the shproj file instead.